articles/Glossary.md

```diff
@@ -23,7 +23,7 @@ uid: microsoft.quantum.glossary
 | <a id="global-phase"></a>Global phase | Two states that are identical up to a multiple of a complex number $e^{i\phi}$ are said to differ up to a global phase. Unlike local phases, global phases cannot be observed through any measurement. See [Pauli measurements](xref:microsoft.quantum.concepts.pauli) for more details. |
 |Measurement|Obtaining a classical bit from a qubit (or set of qubits). See the [Qubit Concepts](xref:microsoft.quantum.concepts.qubit) section for more details.|
 |Mutable|A variable whose value may be changed after it is created.|
-|Namespace|A label for a collection of related names (typically operations, functions, and types). For instance the namespace `Microsoft.Quantum.Canon` labels all of the symbols defined by [the canon](xref:microsoft.quantum.canon).|
+|Namespace|A label for a collection of related names (typically operations, functions, and types). For instance the namespace `Microsoft.Quantum.Standard` labels all of the symbols defined by [the Standard](xref:microsoft.quantum.canon).|
```

Comment:
Microsoft.Quantum.Standard is the name of the NuGet package that includes all of the standard libraries,  including `Microsoft.Quantum.Canon`, `Microsoft.Quantum.Math`, and `Microsoft.Quantum.Preparation`.
```suggestion
|Namespace|A label for a collection of related names (typically operations, functions, and types). For instance the namespace [`Microsoft.Quantum.Preparation`](xref:microsoft.quantum.preparation) labels all of the symbols defined in the standard library that help with preparing initial states.|
```